website/blog/2024-06-24-release-1.11.md

```diff
@@ -0,0 +1,249 @@
+- fix: use podman machine inspect to know if machine is rootful by @benoitf in [#7024](https://github.com/containers/podman-desktop/pull/7024)
+- fix: make markdown TOC links clickable by @benoitf in [#7010](https://github.com/containers/podman-desktop/pull/7010)
+- fix: increasing default timeout for extension activation by @axel7083 in [#7053](https://github.com/containers/podman-desktop/pull/7053)
+- fix: show description link on preflight checks on dashboard by @lstocchi in [#7056](https://github.com/containers/podman-desktop/pull/7056)
+- fix: edit button should be visible during started and stopped by @benoitf in [#7063](https://github.com/containers/podman-desktop/pull/7063)
+- fix: add tests for PreflightChecks by @lstocchi in [#7069](https://github.com/containers/podman-desktop/pull/7069)
+- fix(Button): remove unused `hidden` property by @axel7083 in [#7092](https://github.com/containers/podman-desktop/pull/7092)
+- fix: flaky test port availability by @axel7083 in [#7110](https://github.com/containers/podman-desktop/pull/7110)
+- fix: reset initialize and start mode after starting provider by @lstocchi in [#7115](https://github.com/containers/podman-desktop/pull/7115)
+- fix: stop informer not more valid and handle similar contexts by @lstocchi in [#6934](https://github.com/containers/podman-desktop/pull/6934)
+- fix: custom extension install modal buttons by @axel7083 in [#7067](https://github.com/containers/podman-desktop/pull/7067)
+- fix(tests): adding label to avoid duplicates by @axel7083 in [#7160](https://github.com/containers/podman-desktop/pull/7160)
+- fix: provider card UI on dashboard by @lstocchi in [#7082](https://github.com/containers/podman-desktop/pull/7082)
+- fix(extension-loader): cleanup resource on extension error by @axel7083 in [#7228](https://github.com/containers/podman-desktop/pull/7228)
+- fix: remove tinro from Tab and ui by @lstocchi in [#7288](https://github.com/containers/podman-desktop/pull/7288)
+- fix: remove tinro dependency from SettingsNavItem component by @axel7083 in [#7280](https://github.com/containers/podman-desktop/pull/7280)
+- fix: button should have spinner when inProgress is enable by @axel7083 in [#7259](https://github.com/containers/podman-desktop/pull/7259)
+- fix: table child selection by @deboer-tim in [#7555](https://github.com/containers/podman-desktop/pull/7555)
+- fix: close watcher when FileSystemWatcher is disposed by @feloy in [#7503](https://github.com/containers/podman-desktop/pull/7503)
+- fix: table properly support undefined value result from rendermapping by @axel7083 in [#7723](https://github.com/containers/podman-desktop/pull/7723)
+- fix(UI): formpage missing content shadow by @axel7083 in [#7733](https://github.com/containers/podman-desktop/pull/7733)
+
+---
+
+## Other Chores
+
+We also made various other improvements and updates, including:
```

Comment:
Chores are usually really minor, internal, or setup for one of the features above. IMHO we should just say the total number of issues resolved or have a link if people are really interested in one of the previous sections, no details. This just bloats the release notes for something nobody will read.